docs/whatsnew/3.0.rst

```diff
@@ -12,20 +12,79 @@ The SunPy project is pleased to announce the 3.0 release of the sunpy package.
 
 On this page, you can read about some of the big changes in this release:
 
+* :ref:`whatsnew-3.0-map-plotting`
+* :ref:`whatsnew-3.0-map-eui`
+* :ref:`whatsnew-3.0-map-metadata`
+* :ref:`whatsnew-3.0-instr`
 * :ref:`whatsnew-3.0-python`
 
 SunPy 3.0 also includes a large number of smaller improvements and bug fixes, which are described in the :ref:`changelog`.
 
 By the numbers:
 
-* X commits have been added since 2.0
-* X issues have been closed since 2.0
-* X pull requests have been merged since 2.0
-* X people have contributed since 2.0
+* X commits have been added since 2.1
+* X issues have been closed since 2.1
+* X pull requests have been merged since 2.1
+* X people have contributed since 2.1
 * X of which are new contributors
 
 Please find below a selection of what we consider to be the biggest changes or features with this release.
 
+.. _whatsnew-3.0-map-plotting:
+
+Improvements to Visualization of Maps
+=====================================
+
+In this release a number of Map visualisation methods have been improved to be more coordinate aware than previous releases.
+The first new feature is :meth:`sunpy.map.GenericMap.draw_quadrangle` which replaces the old ``draw_rectangle`` method.
+:meth:`~sunpy.map.GenericMap.draw_quadrangle` draws rectangles where the edges follow lines of constant longitude and latitude in the coordinate system in which the box is drawn.
+
+.. minigallery:: sunpy.map.GenericMap.draw_quadrangle
+
+The next change is to :meth:`sunpy.map.GenericMap.draw_limb`, which now supports drawing the limb as seen by any observer on any image.
+This means that it is possible to visualize the sections of the limb which are visible in one map on another, or to draw the limb as seen from helioprojective map observer on a synoptic map.
+
+.. minigallery:: sunpy.map.GenericMap.draw_limb
```

Comment:
This is generating a way too large number of examples and almost none of them are showing off the new functionality. @ayshih any ideas?